spec/spec_helper_acceptance.rb

```diff
@@ -108,5 +109,16 @@
       create_remote_file(puppetserver, '/etc/puppetlabs/code/environments/production/manifests/site.pp', '')
       on puppetserver, "chmod 0644 /etc/puppetlabs/code/environments/production/manifests/site.pp"
     end
+
+    on setup_nodes, puppet("resource package puppet-bolt ensure=installed")
+    bolt_cfg = <<-EOS
+modulepath: "/etc/puppetlabs/code/modules:/etc/puppetlabs/code/environments/production/modules"
+ssh:
```

Comment:
Document this is because of bolt

Reply:
Comment added at top, rest is pretty obvious based on variable names and having "bolt" in commands".